CONTRIBUTING.md

```diff
@@ -42,6 +42,12 @@ yarn slicemachine:dev
 Now you can go to your browser and open `localhost:9999` to see the Slice Machine app up and running with some demo slices and types.
 You're now ready to start working on your changes and see them updated in the app.
 
+## End-to-End tests
+
+In order to contribute to Slice Machine you will need to maintain and write new e2e tests:
```

Comment:
Valid options:
- E2E
- e2e
- End-to-End
- end-to-end


Both are used sometimes at the same time in doc, let's choose:
- E2E
- e2e

Since it's better anyway for yarn scripts.